ios-base/build_kotlin_modules.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "include ':ios-combined'" > $SRCROOT/../ios-settings.gradle
```

Comment:
If you would like to include this project only during ios build, there is another solution. At first, please add the following to settings.gradle.

```
if (System.getenv("BUILD_IOS") == "true") {
  include ':ios-combined'
}
```

and then, run `BUILD_IOS=true ./gradlew ios-combined:xxx`. ios-combined would be resolved only while BUILD_IOS=true was supplied.

Reply:
🙇

Reply:
Is this solution for only represents
```
echo "include ':ios-combined'" > $SRCROOT/../ios-settings.gradle
```
line? Does this solution mean the line can be removed?

Reply:
Yeah, exactly.

Reply:
I see. OK. I will use that solution.

Reply:
I've done above. 👍 https://github.com/DroidKaigi/conference-app-2020/pull/447/commits/251f9da59d936c57a3ad464b6b8490e68b586579